wellcomecollection/wellcomecollection.org
# Highlight tour transcripts

For #11038 

## What does this change?
Adding the video subtitles and audio transcripts to the exhibition guide stop pages using the existing `CollapsibleContent` component

<img width="699" alt="image" src="https://github.com/user-attachments/assets/7bc981d4-e07f-4f4c-ae25-64dc39d69a2a">

## How to test
- Run locally with the Prismic stage respository url (`wellcomecollection-stage`)
- Visit an [audio guide stop page](http://localhost:3000/guides/exhibitions/ZrHvtxEAACYAWmfc/audio-without-descriptions/2)
- Verify the 'Read audio transcript' link works
- Do the same for the [video version](http://localhost:3000/guides/exhibitions/ZrHvtxEAACYAWmfc/bsl/2)
- Audio player should be sticky to beneath the header/video player should scroll off the screen)

## How can we measure success?
We don't currently have tracking on the CollapsibleContent component. Do we want to add any?

## Have we considered potential risks?
Can't think of any
